.github/workflows/ci.yml

```diff
@@ -91,3 +91,22 @@ jobs:
       - name: Cleanup cache
         if: steps.cache.outputs.cache-hit != 'true'
         run: cargo cache -a
+  cargo-deny:
```

Comment:
Github still complains about this part: https://github.com/Mic92/vmsh/actions/runs/603024902